README.md

```diff
@@ -1,237 +1,122 @@
-External Dependency Manager for Unity
-========
+# External Dependency Manager for Unity
 
-# Overview
+[![openupm](https://img.shields.io/npm/v/com.google.external-dependency-manager?label=openupm&registry_uri=https://package.openupm.com)](https://openupm.com/packages/com.google.external-dependency-manager/)
+[![openupm](https://img.shields.io/badge/dynamic/json?color=brightgreen&label=downloads&query=%24.downloads&suffix=%2Fmonth&url=https%3A%2F%2Fpackage.openupm.com%2Fdownloads%2Fpoint%2Flast-month%2Fcom.google.external-dependency-manager)](https://openupm.com/packages/com.google.external-dependency-manager/)
 
-The External Dependency Manager for Unity (EDM4U)
-(formerly Play Services Resolver / Jar Resolver) is intended to be used by any
-Unity plugin that requires:
+## Overview
 
-   * Android specific libraries (e.g
-     [AARs](https://developer.android.com/studio/projects/android-library.html)).
-   * iOS [CocoaPods](https://cocoapods.org/).
-   * Version management of transitive dependencies.
-   * Management of Package Manager (PM) Registries.
+The External Dependency Manager for Unity (EDM4U) (formerly Play Services
+Resolver/Jar Resolver) is intended to be used by any Unity package or user that
+requires:
 
-Updated releases are available on
-[GitHub](https://github.com/googlesamples/unity-jar-resolver)
-
-# Background
-
-Many Unity plugins have dependencies upon Android specific libraries, iOS
-CocoaPods, and sometimes have transitive dependencies upon other Unity plugins.
-This causes the following problems:
-
-   * Integrating platform specific (e.g Android and iOS) libraries within a
-     Unity project can be complex and a burden on a Unity plugin maintainer.
-   * The process of resolving conflicting dependencies on platform specific
-     libraries is pushed to the developer attempting to use a Unity plugin.
-     The developer trying to use your plugin is very likely to give up when
-     faced with Android or iOS specific build errors.
-   * The process of resolving conflicting Unity plugins (due to shared Unity
-     plugin components) is pushed to the developer attempting to use your Unity
-     plugin. In an effort to resolve conflicts, the developer will very likely
-     attempt to resolve problems by deleting random files in your plugin,
-     report bugs when that doesn't work and finally give up.
-
-EDM provides solutions for each of these problems.
-
-If, after reading this, you need to debug usage of EDM in a Unity project,
-consult the debugging [step-by-step guide](troubleshooting-faq.md).
-
-## Android Dependency Management
-
-The *Android Resolver* component of this plugin will download and integrate
-Android library dependencies and handle any conflicts between plugins that share
-the same dependencies.
+*   Android specific libraries (e.g
```

Comment:
The formatting of the stars is off, I believe it needs to be the single space for the markdown to render it correctly.

Reply:
I doublechecked that they render correctly on GitHub - see https://github.com/f1yingbanana/unity-jar-resolver for the preview on my fork.